src/bundle/Resources/public/js/scripts/fieldType/ezdatetime.js

```diff
@@ -91,7 +93,8 @@
         const sourceInput = field.querySelector(SELECTOR_INPUT);
         const flatPickrInput = field.querySelector(SELECTOR_FLATPICKR_INPUT);
         const btnClear = field.querySelector('.ez-data-source__btn--clear-input');
-        const defaultDate = sourceInput.value ? new Date(sourceInput.value * 1000) : null;
+        const defaultDateWithUserTimezone = eZ.helpers.timezone.convertDateToTimezone(sourceInput.value * 1000);
+        const defaultDate = sourceInput.value ? new Date(defaultDateWithUserTimezone.format('YYYY-MM-DD HH:mm:ss')) : null;
```

Comment:
Why this format is hardcoded? Seems like should be from the config.

Reply:
I wanted to get date of type _Date_ with given year, day, month, hour etc. but with browser timezone, which is used in _flatpickr_. 
Ref: https://stackoverflow.com/questions/29132096/momentjs-todate-timezone-gets-reset#answer-29170326

Maybe there is better way of doing this.

Reply:
In the helper there is method `formatDate` - will format without timezone as far as I know.

Reply:
But I would also have to pass `YYYY-MM-DD HH:mm:ss` to this `formatDate` function, because e.g. _userPreferedFullDateFormat_ may not include seconds. Given this, are there any benefits of using `formatDate` from timezone helper?

Reply:
And for me, downside of `formatDate` is that this formatter may change in the future as it has changed recently: https://github.com/ezsystems/ezplatform-admin-ui/commit/ef673a1c1ac8403c54c2381d9323296af50f2c9b#diff-7176012c51b2ab10ab527e44f38c1544R10
so I don't think it is safe to use it when you want to have full date without timezone. I think that `formatDate` should be used only for formatting date when displaying it to user.

Reply:
As I understand this is the date that will be displayed in the input. It should be consistent with other part of the application